Paper.tex

```diff
@@ -985,22 +985,23 @@ \section{Block Finalisation} \label{ch:finalisation}
 
 \subsection{Ommer Validation}
 
-The validation of ommer headers means nothing more than verifying that each ommer header is both a valid header and satisfies the relation of $N$th-generation ommer to the present block where $N \leq 6$. The maximum of ommer headers is two. Formally:
+The validation of ommer headers means nothing more than verifying that each ommer header is both a valid header and satisfies the relation of $N$th-generation ommer to the present block where $N \leq 6$. The maximum of ommer headers is two. An ommer may be included only in one block in a blockchain. Formally:
```

Comment:
Is there a list of all ommers that have been included in any block in a blockchain?  How is it enforced?

Reply:
So should this PR be closed?

Reply:
An ommer (uncle) is a block header with a `parentHash` and it enforced by comparing this with the block's parent hash.
https://github.com/ethereum/go-ethereum/blob/master/consensus/ethash/consensus.go#L211